pants.ci.toml

```diff
@@ -3,4 +3,7 @@ dynamic_ui = false
 colors = true
 
 [pytest]
-args = ["-vv"]
\ No newline at end of file
+args = ["-vv"]
+
+[sendwave-docker]
+cache_from = "us-docker.pkg.dev/sendwave-development/sendwave-docker/pants-docker"
```

Comment:
Not sure if this path is safe to store in a public repo? What do you think?

Reply:
We discussed async and decided to remove it once we've confirmed that the build cache works on CI.